filters/austerity.go

```diff
@@ -1,15 +1,14 @@
-package main
+package filters
 
 import (
 	"fmt"
 	"math"
 	"math/rand"
 
-	"github.com/stripe/unilog"
 	"github.com/stripe/unilog/clevels"
 )
 
-func austerityFilter(line string) string {
+func AusterityFilter(line string) string {
```

Comment:
Just calling out: one thing that's weird about this shift is that AusterityFilter is now an exported function, and calling it will block *unless* a different function (in a third package) is called first.

Is there a way we can make this less of a footgun? Perhaps with sync.Once, to ensure that the updating goroutine has been kicked off?

Reply:
I see, makes sense. Moved the `clevels.SendSystemAusterityLevel` call into the filter wrapped in a `sync.Once`. Also added a test (backported from the internal version) to make sure this doesn't break anything.